src/twisted/newsfragments/6670.misc

```diff
@@ -0,0 +1,3 @@
+The --https argument to web plugin is now deprecated
+It is possible to give the web plugin the --port argument
```

Comment:
Looks like this also deprecates `--port`.  Should this be `--http`?

Reply:
Good catch! I changed this between writing the newsfragment and submitting the PR, since this was the suggestion in the ticket.